database/seeders/AccessibilitiesSeederHrms.php

```
@@ -948,6 +948,13 @@ public function run(): void
                     'created_at' => Carbon::now(),
                     'updated_at' => Carbon::now(),
                 ],
+                [
+                    'id' => 420,
+                    'accessibilities_name' => AccessibilityHrms::HRMS_REPORTS_USERACCESSIBILITY->value,
+                    'deleted_at' => null,
+                    'created_at' => Carbon::now(),
```

PHPMD (2.15.0)

Avoid using static access to class '\Carbon\Carbon' in method 'run'. (Clean Code Rules)

(StaticAccess)